### DISCUSSION TITLE
Ignored apps by yabai can't be selected on the same screen but are automatically selected when moving from other display

### DISCUSSION THREAD

**Discussion by Demianeen** [asmvik/yabai#2059] [Q&A] [answered]
I have two default window focus commands as well as ignore apps setting:
```
# moving between windows focus within a space
alt - j : yabai -m window --focus south
alt - k : yabai -m window --focus north
alt - h : yabai -m window --focus west
alt - l : yabai -m window --focus east

# moving between displays
ralt - a : yabai -m display --focus west
ralt - s : yabai -m display --focus east

yabai -m rule --add app="^Simulator$" manage=off
```

I found the issue when I try to move between two displays that the simulator is selected despite it having the manage option to off:

https://github.com/koekeishiya/yabai/assets/51330172/b6654cdf-a03f-42c9-bbf4-f7899b0e354c

Do you know if this is intended behaviour? Is it possible to do something about it?

**Comment by asmvik** [comment, marked as answer]
This is working as intended. If you want to focus a non-managed window you need to use the window-id, not a cardinaø direction.
